Integrations docs (#2808)
mattermost/docs: source/deployment/incident-response-and-monitoring.rst

Incident Response and Monitoring Tools
============================================

Our `Integrations Directory <https://integrations.mattermost.com>`_ has several integrations to connect incident response and monitoring tools in Mattermost.

Here are some popular options below, including self-hosted on-prem or self-hosted private cloud solutions, and vendor-hosted SaaS solutions.

AWS CloudWatch SNS
~~~~~~~~~~~~~~~~~~~~~~~~

 - Two-way integration between Mattermost and Amazon AWS SNS, developed by Carlos Tadeu Panato Junior, supported by Mattermost.
 - Receive SNS notifications from Alerts created by Amazon AWS CloudWatch and sent via AWS SNS.

>  - Get operational data and actionable insights to monitor Mattermost in order to respond to system-wide performance changes.

Reviewer: Is it 'performance changes'?

Reviewer 2: Changed text to become "performance changes system-wide"

 - Source code + docs: https://github.com/mattermost/mattermost-plugin-aws-SNS

PagerDuty
~~~~~~~~~~~~~~~~~~~~~~~~

 - Get automatic information regarding the state of an incident to Mattermost, developed by PagerDuty.

>  - Receive all incident updates for any PagerDuty Service in any Mattermost channel to optimize incident resolution time and process.

Reviewer: ```suggestion
 - Send incident updates for any PagerDuty Service to any Mattermost channel to optimize incident resolution time and process.
```

 - Docs: https://www.pagerduty.com/docs/guides/mattermost-integration-guide/

Splunk
~~~~~~~~~~~~~~~~~~~~~~~~

 - Monitor a Mattermost environment.

>  - Receive application performance insights in any Mattermost channel for better troubleshooting and monitoring and to increase uptime.

Reviewer: ```suggestion
 - Send application performance insights to any Mattermost channel for better troubleshooting and monitoring, and to increase system uptime.
```
